constellations/gemini-user/src/afh/afh.js

```diff
@@ -0,0 +1,152 @@
+"use strict";
+require("./afh.sass");
+import React from "react";
+import API from "../utils/api.js";
+import moment from "moment";
+
+export class AFHPage extends React.Component {
+    state = {
+        currentTab: "Math",
```

Comment:
Is there a way we can keep the value "behind the scenes" lower case `math` but still keep the user-facing display "Math", "English", etc.? Typically for inner values, you don't want to avoid having capitalized values because it could introduce discrepancies.

You could define a Javascript object above that looks like this:

const subjectDisplayNames = {
    math: "Math",
    english: "English",
    computer: "Computer Programming"
}